src/Zip_TEST.cc

```diff
@@ -16,7 +16,7 @@
 */
 
 // All these helper functions have been copied from
-// https://github.com/ignitionrobotics/ign-common/raw/master/src/Filesystem_TEST.cc
+// https://github.com/ignitionrobotics/ign-common/raw/ign-common3/src/Filesystem_TEST.cc
```

Comment:
```suggestion
// https://github.com/ignitionrobotics/ign-common/raw/ign-common4/src/Filesystem_TEST.cc
```

Reply:
I think this one is fine, because it's pointing to the original place where the functions were copied from.